src/platform/linker/loadall.c

```diff
@@ -20,27 +20,53 @@
 #include <assert.h>
 #include <sys/mman.h>
 
+char *service_name = NULL;
+
+struct private_symtab{
+	char name[MAX_SYMB_LEN];
+	unsigned long vma_val;
+};
+
 unsigned long getsym(bfd *obj, char* symbol)
 {
         long storage_needed;
-        asymbol **symbol_table;
+        asymbol **symbol_table = NULL;
+	static struct private_symtab *private_symtabs;
```

Comment:
Inconsistent use of spaces/tabs.  This file is incorrectly spaces, but you should at least be consistent with yourself.

Reply:
Oops, thank you. I'll update the file to use tabs throughout.